unison-cli/src/Unison/CommandLine/Completion.hs

```diff
@@ -0,0 +1,321 @@
+{-
```

Comment:
Much of this module was just collected from other locations. I'll note the new stuff.